translate/public/translate.html

```diff
@@ -16,15 +16,18 @@
     <title>Pontoon</title>
 
     <!-- Inlining CSS mitigates/prevents fouc. -->
-    <style> body { background: #3f4752 !important; } </style>
+    <style> body.dark-theme { background: #3f4752 !important; } </style>
     <link href="/static/css/fontawesome-all.css" rel="stylesheet" />
     <link href="/static/css/boilerplate.css" rel="stylesheet" />
     <link href="/static/css/fonts.css" rel="stylesheet" />
+    <link href="/static/css/dark-theme.css" rel="stylesheet" />
+    <link href="/static/css/light-theme.css" rel="stylesheet" />
```

Comment:
Why not include these files in PIPELINE_CSS? We already include dark-theme.css there.